public/javascript/posts/BookAppointmentModal.js

```diff
@@ -78,6 +80,16 @@ angular.module('BookAppointmentApp')
       { "id": 4, "label": "OPT Schedule"},
       { "id": 5, "label": "Inscheduled Emergencies"}
     ];
+    if($scope.selected_event.event_type == 'booking' && $scope.selected_event.prepay_amount > 0){
+      if($scope.selected_event.is_booked){
```

Comment:
please call this flag `is_paid` as this is the flag which will be received from the server